### PR TITLE
[Doc] Change Vimeo video for YouTube

### PR DESCRIPTION
**Pull request by erwanMarmelab**
## Problem

Vimeo will delete our e-commerce video.

## Solution

Change publication platform for YouTube

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ A frontend Framework for building single-page applications running in the browse
 
 [Home page](https://marmelab.com/react-admin/) - [Documentation](https://marmelab.com/react-admin/documentation.html) - [Examples](#examples) - [Blog](https://marmelab.com/en/blog/#react-admin) - [Releases](https://github.com/marmelab/react-admin/releases) - [Support](#support)
 
-[![react-admin-demo](https://marmelab.com/react-admin/img/react-admin-demo-still.png)](https://vimeo.com/474999017)
+[![react-admin-demo](https://marmelab.com/react-admin/img/react-admin-demo-still.png)](https://www.youtube.com/watch?v=gW6BvodBIzE&ab_channel=react-admin)
```
Comment (erwanMarmelab) [inline README.md:7]:
```suggestion
[![react-admin-demo](https://marmelab.com/react-admin/img/react-admin-demo-still.png)](https://www.youtube.com/watch?v=gW6BvodBIzE
```

File: packages/react-admin/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ A frontend Framework for building data-driven applications running in the browse
 
 [Home page](https://marmelab.com/react-admin/) - [Documentation](https://marmelab.com/react-admin/Tutorial.html) - [Demo](https://marmelab.com/react-admin-demo/) - [Blog](https://marmelab.com/en/blog/#react-admin) - [Releases](https://github.com/marmelab/react-admin/releases) - [Support](https://stackoverflow.com/questions/tagged/react-admin)
 
-[![react-admin-demo](https://marmelab.com/react-admin/img/react-admin-demo-still.png)](https://vimeo.com/474999017)
+[![react-admin-demo](https://marmelab.com/react-admin/img/react-admin-demo-still.png)](https://www.youtube.com/watch?v=gW6BvodBIzE&ab_channel=react-admin)
```
Comment (erwanMarmelab) [inline packages/react-admin/README.md:7]:
```suggestion
[![react-admin-demo](https://marmelab.com/react-admin/img/react-admin-demo-still.png)](https://www.youtube.com/watch?v=gW6BvodBIzE)
```

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ A frontend Framework for building single-page applications running in the browse
 
 [Home page](https://marmelab.com/react-admin/) - [Documentation](https://marmelab.com/react-admin/documentation.html) - [Examples](#examples) - [Blog](https://marmelab.com/en/blog/#react-admin) - [Releases](https://github.com/marmelab/react-admin/releases) - [Support](#support)
 
-[![react-admin-demo](https://marmelab.com/react-admin/img/react-admin-demo-still.png)](https://vimeo.com/474999017)
+[![react-admin-demo](https://marmelab.com/react-admin/img/react-admin-demo-still.png)](https://www.youtube.com/watch?v=gW6BvodBIzE
```
Comment (fzaninotto) [inline README.md:7]:
```suggestion
[![react-admin-demo](https://marmelab.com/react-admin/img/react-admin-demo-still.png)](https://www.youtube.com/watch?v=gW6BvodBIzE)
```

File: docs/Demos.md [hunk 4]
```diff
@@ -208,7 +208,9 @@ If you want to see what react-admin is capable of, or if you want to learn from
 
 The admin of a fictional poster shop, allowing to manage sales, products, customers and reviews. Built by the core team.
 
-<div style="padding:56.25% 0 0 0;position:relative;"><iframe src="https://player.vimeo.com/video/474999017?h=4557e43b02&title=0&byline=0&portrait=0" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" allow="autoplay; fullscreen; picture-in-picture" allowfullscreen></iframe></div>
+<div style="padding:56.25% 0 0 0;position:relative;">
+<iframe width="1264" height="711" src="https://www.youtube.com/embed/gW6BvodBIzE" title="react admin e commerce demo" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (fzaninotto) [inline docs/Demos.md:212]: use the nocookie domain (see other youtube embeds)

**Review by fzaninotto — CHANGES_REQUESTED** [review]
Missing replacements:

- data generator readme
- demo example readme
And many more (look for 'vimeo' in the entire code)

**Comment by erwanMarmelab** [comment]
> Missing replacements:
> 
> * data generator readme
> * demo example readme
>   And many more (look for 'vimeo' in the entire code)

I don't agree. Those Vimeo videos refers to other videos and not [this video](https://vimeo.com/474999017) (cf https://vimeo.com/user3950371)

**Comment by fzaninotto** [comment]
> I don't agree. Those Vimeo videos refers to other videos and not [this one](https://vimeo.com/474999017) (cf [all videos we have](https://vimeo.com/user3950371))

You're right, but the problem is even worse: the other video is a demo of react-admin v2, so it's even more outdated!

**Pull request merged**